groups/ecs-service/locals.tf

```diff
@@ -4,7 +4,7 @@ locals {
   name_prefix               = "${local.stack_name}-${var.environment}"
   service_name              = "docs-developer"
   container_port            = "8080" # default tomcat port required here until prod docker container is built allowing port change via env var
-  docker_repo               = "local/docs.developer.ch.gov.uk" # using local repo here until prod ready repo and contaner image is created
+  docker_repo               = "docs.developer.ch.gov.uk"
```

Comment:
Should this be upodated to reference the new `var.docker_repo`?

Reply:
No Billy wanted the repo hardcoded so no one can mistakenly deploy from another repo